init_venv.sh

```diff
@@ -110,11 +110,12 @@ if [[ -z $CUDA_VERSION_CODE ]]; then
   echo torch==${TORCH_VERSION}+cpu >> ${CONSTRAINTS_FILE}
   echo torchvision==${TORCHVISION_VERSION}+cpu >> ${CONSTRAINTS_FILE}
 elif [[ $CUDA_VERSION_CODE == "102" ]]; then
-  pip install torch==${TORCH_VERSION} torchvision==${TORCHVISION_VERSION} -c ${CONSTRAINTS_FILE} || exit 1
+  pip install torch==${TORCH_VERSION}+cu${CUDA_VERSION_CODE} torchvision==${TORCHVISION_VERSION}+cu${CUDA_VERSION_CODE} -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html \
```

Comment:
Hm...

1. We had two separate branches for CUDA 10.2 and any other (including 11.1) because PyPI repos and package names were different. If this is not the case anymore, they can be merged.
2. Having `1.8` explicitly mentioned in the URL removes the flexibility and will break for PyTorch 1.9.0.
3. Should the PyPI repo for CPU-only build be changed as well?